packages/pools/src/common.ts

```diff
@@ -0,0 +1,30 @@
+import { Dec, Int } from "@keplr-wallet/unit";
```

Comment:
Regarding this file, have you looked at `weighted.ts` in the `/math` package? Perhaps this would fit well there.

Reply:
Yeah Agreed.